readme.md

```diff
@@ -25,12 +25,14 @@ You've written the following HTML. When you look at it in your browser, it's jus
 ```
 
 > Your answer...
+There is no closing tag for the <title> in the <head>.
 
 ## Question 2
 
 What's the purpose of the `alt` attribute on image tags?
 
 > Your answer...
+So people who can't see but have special programs that read things out loud to them can know what the content of the picture is.
```

Comment:
This is the primary reason... I'd also note that it enables search engine crawlers to gain more information about what an image is about, for SEO. (Think google image search)